cucumber/api/features/step_definitions/authz_steps.rb

```diff
@@ -103,3 +104,10 @@
   grantee = lookup_user(grantee)
   grantor.grant_to grantee
 end
+
+Given(/^I grant group "([^"]*)" to (user|host) "([^"]*)"$/) do |group_id, role_type, grantee|
```

Comment:
Are you anticipating this will be used in the future? B.c rn this is not used in codebase

Reply:
It is used in another PR for authn-azure tests